app/views/profiles/edit.html.haml

```diff
@@ -25,7 +25,7 @@
           %p
             = f.label :bio
             = f.text_area :bio, class: 'form-control', placeholder: 'Enter your bio', rows: 7, maxlength: 500
-            %p.help-block Bio is limited to 500 characters.
+            %p.help-block.red-text Bio is limited to 500 characters.
```

Comment:
let's revert this to the green version since it's not necessarily an error message but a true hint.